.github/workflows/lint.yml

```diff
@@ -0,0 +1,29 @@
+name: LINT
+on: [push]
+jobs:
+  build:
+    name: lint and format
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node: ['14.x']
+        os: [ubuntu-latest, windows-latest, macOS-latest]
```

Comment:
hmm is there a reason to run on all these os's?

Reply:
Nope, this is usually done when building libraries. Let me stick with one os.